tests/integ/test_experiments_analytics.py

```diff
@@ -36,7 +36,15 @@ def experiment(sagemaker_session):
                 TrialComponentName=trial_component_name, TrialName=trial_name
             )
 
-        time.sleep(15)  # wait for search to get updated
```

Comment:
I would suggest keeping this initial sleep in place, since otherwise you'll allow no time for indexing

Reply:
I moved the initial sleep inside the loop to increase it 3 times. Does that allow time for indexing?